src/js/labels/type.ts

```diff
@@ -0,0 +1,17 @@
+/* Smaller label object for representing labels in a list */
```

Comment:
```suggestion
/*A partial label including essentials for rendering a list of labels*/
```
My bad. In general it is redundant to list types in the description